tests/functional/lib/BackbeatClient.js

```diff
@@ -83,4 +64,51 @@ describe('BackbeatClient unit tests with mock server', () => {
             return done();
         });
     });
+
+    it('should get bucket metadata', done => {
+        const destReq = backbeatClient.getBucketMetadata({
+            Bucket: bucketName,
+        });
+        return destReq.send((err, data) => {
+            assert.ifError(err);
+            assert.strictEqual(typeof data.acl, 'object');
+            assert.strictEqual(typeof data.name, 'string');
+            assert.strictEqual(typeof data.owner, 'string');
+            assert.strictEqual(typeof data.ownerDisplayName, 'string');
+            assert.strictEqual(typeof data.creationDate, 'string');
+            assert.strictEqual(typeof data.mdBucketModelVersion, 'number');
+            assert.strictEqual(typeof data.transient, 'boolean');
+            assert.strictEqual(typeof data.deleted, 'boolean');
+            assert.strictEqual(typeof data.uid, 'string');
+            return done();
+        });
+    });
+
+    it('should get object list', done => {
+        const destReq = backbeatClient.getObjectList({
+            Bucket: bucketName,
+        });
+        return destReq.send((err, data) => {
+            assert.ifError(err);
+            data.Contents.forEach(object => {
```

Comment:
Same thing here, use build the expected object and assert using `deepStrictEqual`